operations/rates.md

```diff
@@ -136,7 +136,48 @@ The price in the response is dependent on the enterprise's [pricing](configurati
 ```javascript
 {
     "Currency": "EUR",
-    "BasePrices": [ 20, 20, 20 ],
+    "BaseAmountPrices": [
```

Comment:
`BasePrices` is being replaced by `BaseAmountPrices` - this is a deprecation and therefore an entry must be added to the **Deprecations** table; please also include the **discontinued** date which should be +6 months for a minor breaking change or +12 months for a major breaking change :-)